Packs/ServiceNow/Scripts/ServiceNowTroubleshoot/README.md

```diff
@@ -0,0 +1,29 @@
+Troubleshot ServiceNow v2 integration by displaying health information of enabled instances and old and active incidents of disabled instances.
```

Comment:
Troubleshoot the ServiceNow v2 integration by displaying the health information of enabled instances along with old and active incidents of disabled instances.